tests/cpp/test_persistent_buffer.cpp

```diff
@@ -1456,4 +1457,121 @@ TEST_F(PersistentBufferTest, InnerPersistentNotEnoughSharedMemory) {
   }
   testValidate(&fusion, outputs, inputs, __LINE__, __FILE__);
 }
+
+using TestParam = std::tuple<DataType, int64_t>;
+using LayerNormSharedMemoryTest = NVFuserFixtureParamTest<TestParam>;
+TEST_P(LayerNormSharedMemoryTest, FusionLayerNormSharedMemoryBuffer_CUDA) {
+  auto [dtype, hidden_size] = GetParam();
+
+  std::unique_ptr<Fusion> fusion_ptr = std::make_unique<Fusion>();
+  Fusion& fusion = *fusion_ptr.get();
+  FusionGuard fg(&fusion);
+  const float kEps = 1e-5;
+  Val* eps_ptr = IrBuilder::create<Val>(kEps);
+  constexpr int64_t dim0 = 2048;
+  std::vector<int64_t> input_shape{dim0, hidden_size};
+  std::vector<int64_t> norm_shape{hidden_size};
+  auto input_half = makeContigTensor(2, dtype);
+  auto weight_half = makeContigTensor(1, dtype);
+  auto bias_half = makeContigTensor(1, dtype);
+  fusion.addInput(input_half);
+  fusion.addInput(weight_half);
+  fusion.addInput(bias_half);
+  auto input = castOp(DataType::Float, input_half);
+  auto weight = castOp(DataType::Float, weight_half);
+  auto bias = castOp(DataType::Float, bias_half);
+  auto result = layer_norm(input, norm_shape, weight, bias, eps_ptr);
+  auto result_output = castOp(dtype, result.output);
+  fusion.addOutput(result_output);
+  fusion.addOutput(result.mean);
+  fusion.addOutput(result.invstd);
+
+  auto options =
+      at::TensorOptions().dtype(data_type_to_aten(dtype)).device(at::kCUDA, 0);
+  at::Tensor aten_input = at::randn(input_shape, options);
+  c10::optional<at::Tensor> aten_weight = at::randn({input_shape[1]}, options);
+  c10::optional<at::Tensor> aten_bias = at::randn({input_shape[1]}, options);
+  std::vector<c10::IValue> aten_inputs = {aten_input, aten_weight, aten_bias};
+
+  // try translate Welford in fusion
+  KernelArgumentHolder runtime_inputs =
+      KernelArgumentHolder::createKernelArgumentHolder(aten_inputs);
+  SegmentCandidateFinder::translateWelfordInFusion(&fusion, runtime_inputs);
+  auto fusion_copy = fusion;
+
+  // check persistent buffer size
+  SchedulerRuntimeInfo runtime_info(
+      &fusion, {aten_input, aten_weight, aten_bias});
```

Comment:
nitpick: can we use `aten_inputs` instead of `{aten_input, aten_weight, aten_bias}` here?